package.json

```diff
@@ -48,10 +48,10 @@
     "govuk-frontend": "^4.6.0",
     "humanize-string": "^3.0.0",
     "js-cookie": "^3.0.5",
-    "maplibre-gl": "^3.3.1",
+    "maplibre-gl": "^3.5.1",
```

Comment:
Needed for https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md#350